sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/MetricsAdvisorAsyncClient.java

```diff
@@ -1508,7 +1510,7 @@ private Mono<PagedResponse<AnomalyAlert>> listAlertsNextPageAsync(
         AlertingResultQuery query = new AlertingResultQuery();
         query.setStartTime(startTime);
         query.setEndTime(endTime);
-        query.setTimeMode(options.getTimeMode());
+        query.setTimeMode(TimeMode.fromString(toStringOrNull(options.getTimeMode())));
```

Comment:
Will `TimeMode.fromString(null)` return "null" (the string) or `null`?

Reply:
It will return null.
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/util/ExpandableStringEnum.java#L59